parser/json/json_test.go

```diff
@@ -36,7 +36,7 @@ func TestJSONParser(t *testing.T) {
 	}
 
 	inputMap := input.(map[string]interface{})
-	if len(inputMap) < 0 {
+	if len(inputMap) <= 0 {
```

Comment:
I think this should actually just be `==`, not `<=`.

Reply:
Oh, sure... I changed it! Thank you!

```len()``` did not return negative values, so I changed it, but I did not check it enough.
I found other similar parts and changed them.